src/main/webapp/WEB-INF/view/StylizedTextParser.java

```diff
@@ -0,0 +1,80 @@
+// StylizedTextParser.java
+// Class with methods to parse a message with markdown elements and
+// convert to HTML stylized text 
+
+//package codeu.controller;
+
+import java.lang.StringBuffer; 
+import java.lang.String; 
+import java.util.Stack; 
+import java.util.Hashtable; 
+
+public class StylizedTextParser {
+
+    /* Maps markdown symbol to array of HTML opening and closing stylized syntax */
+    private Hashtable<Character, String> styleTable; 
+
+    /* constructor */
+    public StylizedTextParser() {
+        styleTable = new Hashtable<Character, String>(); 
+        fill(styleTable); 
+    }
+
+    /* Fills styleTable with markdown elements to be parsed */
+    private static void fill(Hashtable<Character, String> styleTable) {
+        styleTable.put('*', "b");
+        styleTable.put('_', "i");  
+    }
+
+    /* converts markdown to HTML elements */
+    public String parse(String markdownMessage) {
+
+        StringBuffer HTMLMessage = new StringBuffer(); /* holds final HTML message */
```

Comment:
Why do you keep HTMLMessage and Message separate? Can we not just build message with HTML elements?

Reply:
Could you give me an example? Not sure what case you're talking about.

Reply:
"not *bolded" --> not bolded 
"not *bolded" --> not <bold>bolded
Former would  happen using messageBuffer (so don't put the <bold></bold> tags unless a start * and end * is found).